.travis.yml

```diff
@@ -59,6 +63,7 @@ matrix:
       env: TARGET=go
 
 before_install:
+  - ./.travis/before-install-mvn3.3.9-trusty.sh
```

Comment:
Persistence prevails. The reason I didn't dare to put ```dist: trusty``` under ```-os```, was because I'm **smart** enough to use Travis' own validator online: https://lint.travis-ci.org/ against my yaml, and this is the output:

```
in matrix.include section: unexpected key dist, dropping
in matrix.include section: unexpected key osx_image, dropping
in matrix.include section: specified compiler, but setting is not relevant for java
in matrix.include section: unexpected key addons, dropping
in matrix.include.env section: does not support multiple values, dropping CXX=g++-5
```

So I see no reason to fight against them. Anyway, I'll make a PR to @lecode-official again and solve this problem. Thanks for the patience.